CMakeLists.txt

```diff
@@ -136,12 +141,12 @@ FetchContent_Declare(
         range-v3
         #TODO<joka921> Merge this fork into the `qlever` GitHub organisation as soon as it exists.
         GIT_REPOSITORY https://github.com/joka921/range-v3
+
         # This branch removes some differences in the interface between `range-v3` and `std::ranges` s.t.
         # the former can be used as an (almost) drop-in replacement for the latter.
-        GIT_TAG 5ae161451ec1baaac352d7567298d3ac143bccae # branch fork-for-qlever
+        GIT_TAG 42340ef354f7b4e4660268b788e37008d9cc85aa # branch fork-for-qlever
```

Comment:
Note to the reviewers: This is the current version where I have applied several additional fixes to use
`range-v3` more consistently.